src/Dscp/Crypto/ByteArray.hs

```diff
@@ -1,3 +1,6 @@
+{-# LANGUAGE DataKinds                 #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE TypeInType                #-}
```

Comment:
Why did you need these extensions in this file if there's no related changes in the code? :thinking:

Reply:
:man_facepalming: